eras/conway/impl/src/Cardano/Ledger/Conway/CDDL.hs

```diff
@@ -0,0 +1,971 @@
+
+certificate :: Rule
+certificate =
+  "certificate"
+    =:= arr
+      [ a stake_registration
+          / stake_deregistration
+          / stake_delegation
+          / pool_registration
+          / pool_retirement
+          / reg_cert
+          / unreg_cert
+          / vote_deleg_cert
+          / stake_vote_deleg_cert
+          / stake_reg_deleg_cert
+          / vote_reg_deleg_cert
+          / stake_vote_reg_deleg_cert
+          / auth_committee_hot_cert
+          / resign_committee_cold_cert
+          / reg_drep_cert
+          / unreg_drep_cert
+          / update_drep_cert
+      ]
+
+stake_registration :: Named Group
+stake_registration = "stake_registration" =:~ grp [0, a stake_credential]
+
+stake_deregistration :: Named Group
+stake_deregistration = "stake_deregistration" =:~ grp [1, a stake_credential]
+
+stake_delegation :: Named Group
+stake_delegation =
+  "stake_delegation"
+    =:~ grp [2, a stake_credential, a pool_keyhash]
+
+-- POOL
+pool_registration :: Named Group
+pool_registration = "pool_registration" =:~ grp [3, a pool_params]
+
+pool_retirement :: Named Group
+pool_retirement = "pool_retirement" =:~ grp [4, a pool_keyhash, a epoch]
+
+-- numbers 5 and 6 used to be the Genesis and MIR certificates respectively,
+-- which were deprecated in Conway
+
+-- DELEG
+reg_cert :: Named Group
+reg_cert = "reg_cert" =:~ grp [7, a stake_credential, a coin]
+
+unreg_cert :: Named Group
+unreg_cert = "unreg_cert" =:~ grp [8, a stake_credential, a coin]
+
+vote_deleg_cert :: Named Group
+vote_deleg_cert = "vote_deleg_cert" =:~ grp [9, a stake_credential, a drep]
+
+stake_vote_deleg_cert :: Named Group
+stake_vote_deleg_cert =
+  "stake_vote_deleg_cert"
+    =:~ grp [10, a stake_credential, a pool_keyhash, a drep]
+
+stake_reg_deleg_cert :: Named Group
+stake_reg_deleg_cert =
+  "stake_reg_deleg_cert"
+    =:~ grp [11, a stake_credential, a pool_keyhash, a coin]
+
+vote_reg_deleg_cert :: Named Group
+vote_reg_deleg_cert =
+  "vote_reg_deleg_cert"
+    =:~ grp [12, a stake_credential, a drep, a coin]
+
+stake_vote_reg_deleg_cert :: Named Group
+stake_vote_reg_deleg_cert =
+  "stake_vote_reg_deleg_cert"
+    =:~ grp [13, a stake_credential, a pool_keyhash, a drep, a coin]
+
+-- GOVCERT
+auth_committee_hot_cert :: Named Group
+auth_committee_hot_cert =
+  "auth_committee_hot_cert"
+    =:~ grp [14, a committee_cold_credential, a committee_hot_credential]
+
+resign_committee_cold_cert :: Named Group
+resign_committee_cold_cert =
+  "resign_committee_cold_cert"
+    =:~ grp [15, a committee_cold_credential, anchor // VNil]
+
+reg_drep_cert :: Named Group
+reg_drep_cert = "reg_drep_cert" =:~ grp [16, a drep_credential, a coin, anchor // VNil]
+
+unreg_drep_cert :: Named Group
+unreg_drep_cert = "unreg_drep_cert" =:~ grp [17, a drep_credential, a coin]
+
+update_drep_cert :: Named Group
+update_drep_cert = "update_drep_cert" =:~ grp [18, a drep_credential, anchor // VNil]
+
+delta_coin :: Rule
+delta_coin = "delta_coin" =:= VUInt
```

Comment:
`delta_coin` is actually signed
```haskell
delta_coin = "delta_coin" =:= VInt
```

But, it is unused so we can get rid of it. It is unnecessary in the original spec either, so if you can remove it there too that would be helpful.
```suggestion
```